src/scribe_data/cli/main.py

```diff
@@ -30,13 +30,16 @@
 from scribe_data.cli.total import total_wrapper
 from scribe_data.cli.upgrade import upgrade_cli
 from scribe_data.cli.version import get_version_message
+from scribe_data.cli.cli_utils import check_and_update_languages, set_metadata
 
 LIST_DESCRIPTION = "List languages, data types and combinations of each that Scribe-Data can be used for."
 GET_DESCRIPTION = (
     "Get data from Wikidata and other sources for the given languages and data types."
 )
 TOTAL_DESCRIPTION = "Check Wikidata for the total available data for the given languages and data types."
 CONVERT_DESCRIPTION = "Convert data returned by Scribe-Data to different file types."
+UPDATE_DESCRIPTION = "Update the metadata file with available languages and QIDs."
```

Comment:
Main would need the update functionality removed as it'll be covered by the new workflow on PRs :)